### PR TITLE
Implement AI randomly placing ship.

### PR DESCRIPTION
**Pull request by KSARK**
Now AI can follow the player to place the ship in a valid position.

See Issues #7.

### CODE REVIEW

File: src/main/java/cs361/battleships/models/Game.java [hunk 1]
```diff
@@ -49,18 +48,24 @@ public boolean attack(int x, char  y) {
...
     }
 
-    private char randCol() {
-        // TODO implement
-        return 'X';
+    /**
+     * @return char random Column from 'A' to 'J'.
+     */
+    char randCol() {
+        return (char)(new Random().nextInt(10) + 1 + 65);
```
Comment (Aqcurate) [inline src/main/java/cs361/battleships/models/Game.java]: I think this is off by one.

File: src/test/java/cs361/battleships/models/AIPlacementTest.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import static org.junit.Assert.assertFalse;
+
+
+public class AIPlacementTest {
+    @Test
+    public void testInvalidPosition() {
+        Game g = new Game();
+        for (int i = 0; i < 1000; i++) {
+            assertFalse(g.randCol()>'k');
```
Comment (Aqcurate) [inline src/test/java/cs361/battleships/models/AIPlacementTest.java]: randCol() shouldn't be able to return k.

Comment (Aqcurate) [reply src/test/java/cs361/battleships/models/AIPlacementTest.java]: Make sure randCol() doesn't return something less than 'a'.

File: src/test/java/cs361/battleships/models/AIPlacementTest.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            assertFalse(g.randRow()>10);
+            assertFalse(g.randRow()< 0);
```
Comment (Aqcurate) [inline src/test/java/cs361/battleships/models/AIPlacementTest.java]: Change this to less than 1. Rows cannot be zero.

**Comment by Csedge** [comment]
Yes that can help us compelte more,I will complete the placeship please wait me! However we should not return the k

**Review by Csedge — COMMENTED** [review]
Yes that can help us compelte more,I will complete the placeship please wait me! However we should not return the k

**Review by Aqcurate — APPROVED** [review]
Looks good.

**Comment by Csedge** [comment]
That good,let merge it

**Review by staslyakhov — DISMISSED** [review]
Methods should be set to `private`.

**Comment by Aqcurate** [comment]
> Methods should be set to `private`.

Difficult to test private methods. We'll keep it as default permissions for now.

**Pull request merged**